packages/wonder-blocks-toolbar/components/toolbar.js

```diff
@@ -0,0 +1,206 @@
+// @flow
+import * as React from "react";
+import {css, StyleSheet} from "aphrodite";
+
+import {View} from "@khanacademy/wonder-blocks-core";
+
+// TODO(scottgrant): Make sure the toolbars are responsive.
+// import type {MediaSize} from "@khanacademy/wonder-blocks-core";
+
+const dismissIcon =
+    "M12.003 10.588L7.707 6.293a1 1 0 0 0-1.414 1.414l4.295 4.296-4.295 4.295a1 1 0 0 0 1.414 1.414l4.296-4.295 4.295 4.295a1 1 0 0 0 1.414-1.414l-4.295-4.295 4.295-4.296a1 1 0 1 0-1.414-1.414l-4.295 4.295z";
+
+type Props = {
+    /**
+     * A list of nodes to render on the left side of the toolbar. In most
+     * cases, this will be empty.
```

Comment:
Maybe mention that this is used for the close button on modals.

Reply:
Good idea, I updated the comment.